src/CssParsing/StyleRule.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\CssParsing;
+
+/**
+ * This class represents a CSS style rule, including a media query, selectors and a declarations block.
+ *
+ * @internal
+ */
+class StyleRule
```

Comment:
[Definitely the right name I think.](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleRule)